// PlayerNetPackOpcodes.h

HandleOpcode(CMSG_MARKET_ALL_INFO);
HandleOpcode(CMSG_MARKET_COLLECT);
HandleOpcode(CMSG_MARKET_BUY);
HandleOpcode(CMSG_MARKET_BLACK);
HandleOpcode(CMSG_MARKET_TICKET_CHANGE);
HandleOpcode(CMSG_MARKET_MYSTERY_INVEST);
HandleOpcode(CMSG_MARKET_MYSTERY_TAKE_ITEM);

HandleOpcode(CMSG_ACTIVITY_IRON_COLLECT_REWARD);